spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -205,3 +213,62 @@ This approach is depicted in the following Figure that shows an overview of the
   The unbonding operation is completed later once the provider CCV module registered maturity notifications for `VSC2` from all consumer chains.
 
 ![Completion of Unbonding Operations](./figures/ccv-unbonding-overview.png?raw=true)
+
+### Consumer Initiated Slashing
+[&uparrow; Back to Outline](#outline)
+
+For the [Security Model](#security-model) to be preserved, misbehaving validators MUST be slashed (and MAY be jailed, i.e., removed from the validator set). 
+A prerequisite to slash validators is to receive valid evidence of their misbehavior. 
+Thus, when slashing a validator, we distinguish between three events and the heights when they occur:
+- `infractionHeight`, the height at which the misbehavior (or infraction) happened;
+- `evidenceHeight`, the height at which the evidence of misbehavior is received;
+- `slashingHeight`, the height at which the validator is slashed (and jailed). 
+
+> **Note**: In the context of single-chain validation, usually `evidenceHeight = slashingHeight`. 
+
+The [Security Model](#security-model) guarantees that any misbehaving validator can be slashed for at least the unbonding period, 
```

Comment:
I would maybe rephrase this sentence as I understood it in the following way: "If any misbehaving validator misbehaves, it is slashed within the unbonding period".
Maybe: "The security model allows any misbehaving validator to be slashed only within the unbonding period".

Reply:
That's the current behavior in Cosmos SDK. But this is not necessary for security. Validators can be slashed also after the unbonding period elapses and there will be no impact on security. In CCV, this may be the case since a validator must wait to unbond on all consumer chains. Thus, even though the unbonding period elapsed on the provider chain, the validator can still be slashed.